lib/balena.coffee

```diff
@@ -159,6 +160,23 @@ getSdk = (opts = {}) ->
 		get: -> request.interceptors,
 		set: (interceptors) -> request.interceptors = interceptors
 
+	versionHeaderInterceptor =
+		request: (request) ->
+			url = request.url
+
+			if typeof url != 'string'
+				return request
+
+			if typeof request.baseUrl == 'string'
+				url = request.baseUrl + url
+
+			if url.indexOf(opts.apiUrl) == 0
+				request.headers['X-Balena-Client'] = "balena-sdk v#{version}"
```

Comment:
I'm not sure whether it would make more sense as `balena-sdk/1.2.3` similarly to how `node-fetch/1.0` does it :thinking:

Reply:
Yea... I'm changing this to use `/`...